### PR TITLE
dashboard. Use SupportedLanguages client method

### PR DESCRIPTION
**Pull request by dpordomingo**
Blocked by ~https://github.com/bblfsh/sdk/pull/259~, ~https://github.com/bblfsh/bblfshd/pull/167~ and ~https://github.com/bblfsh/client-go/pull/57~

since https://github.com/bblfsh/client-go/pull/57 `bblfsh` languages can be listed using `bblfsh/client-go` through `SupportedLanguages` protocol method so workaround provided by https://github.com/bblfsh/dashboard/pull/122 is no longer needed

### CODE REVIEW

**Comment by dpordomingo** [comment]
Travis fails https://travis-ci.org/bblfsh/dashboard/builds/385760193#L786 throwing from line [server/drivers.go:36](https://github.com/bblfsh/dashboard/pull/133/files#diff-859e0853f2a781e75b1e126b60e71b9fR36)
```shell
server/drivers.go:36:18: cli.NewSupportedLanguagesRequest undefined (type *bblfsh.Client has no field or method NewSupportedLanguagesRequest)
```
It happens because https://github.com/bblfsh/client-go/pull/57 has not been yet merged nor released a newer `v.2.*` version.

(see also https://github.com/bblfsh/client-go/pull/57#issuecomment-393232664)

File: server/drivers.go [hunk 1]
```diff
@@ -22,125 +16,46 @@ type Driver struct {
...
-const DefaultBblfshdCtlPort = 9433
+const driverRepoURLPattern = "https://github.com/bblfsh/%s-driver"
 
 var dockerImageRegExp = regexp.MustCompile("^docker://([^:]+)")
 
-var addressPortRegExp = regexp.MustCompile("^([^:]+):.+")
-
-func (s *Server) handleListDrivers(ctx *gin.Context) {
+func (s *Server) handleSupportedLanguages(ctx *gin.Context) {
 	var req versionRequest
```
Comment (smacker) [inline server/drivers.go]: should be `supportedLanguagesRequest`

**Comment by dpordomingo** [comment]
@smacker your suggestion was addressed by e09ab67
Many thanks for reviewing.
I'll rebase and address your other suggestion about `vendor` dir before merging

**Comment by bzz** [comment]
Great job @dpordomingo  👏 

I guess we need to wait for a new release of bblfsh and go-client and fix the minimal version that support this in our .toml before merging?

**Comment by dennwc** [comment]
Both new releases are up: [client-go@v2.5.0](https://github.com/bblfsh/client-go/releases/tag/v2.5.0), [bblfshd@v2.5.0](https://github.com/bblfsh/bblfshd/releases/tag/v2.5.0).

**Comment by bzz** [comment]
Awesome! Let's fix those and merge, to include this great work in upcoming release

**Comment by smacker** [comment]
@bzz @carlosms  I added more commits to finish it.

We need type in errors because otherwise, successful parse removes error about drivers list if you use old bblfsh.

**Comment by bzz** [comment]
@smacker shall we mention somewhere in README that minimum supported version bblfshd is 2.5.0 now?

**Comment by smacker** [comment]
Added mention of the minimal version in readme.

**Pull request merged**